### DISCUSSION TITLE
How to create a repo from a folder of an existing repo?

### DISCUSSION THREAD

**Discussion by SMerrony** [community/community#21540] [New to GitHub]
<p>I have a repo containing a number of utilities in subdirectories, one of which has grown and now deserves a repo of its own.</p>
<p>Is there any way to extract the folder, and the history of its files, into a new standalone repo?</p>
<p>Thanks.</p>

**Comment by georgiemathews** [comment, marked as answer]
<p>You can follow the steps detailed here: </p>
<p><a href="https://help.github.com/articles/splitting-a-subfolder-out-into-a-new-repository/" rel="nofollow noopener">https://help.github.com/articles/splitting-a-subfolder-out-into-a-new-repository/</a></p>

**Comment by SMerrony** [comment]
<p>Brilliant, thanks - somehow my searches hadn’t turned that up.</p>
